bundles/http/Citrix/Cherwell/Cherwell.js

```diff
@@ -731,31 +734,43 @@ async function getRequest({ dataStore, client }, moduleName, body, endpoint) {
         const data = await resp.json();
         const dataArray = data.records.map(obj => {
             let rObj = JSON.parse(obj);
-            rObj.CreatedDateTime = new Date(obj.CreatedDateTime);
-            rObj.LastModifiedDateTime = new Date(obj.LastModifiedDateTime);
-            rObj.LastModDateTime = new Date(obj.LastModDateTime);
-            rObj.SLAResponseWarning = new Date(obj.SLAResponseWarning);
-            rObj.SLAResolutionWarning = new Date(obj.SLAResolutionWarning);
+            rObj.CreatedDateTime = new Date(rObj.CreatedDateTime);
+            rObj.LastModifiedDateTime = new Date(rObj.LastModifiedDateTime);
+            rObj.LastModDateTime = new Date(rObj.LastModDateTime);
+            rObj.SLAResponseWarning = new Date(rObj.SLAResponseWarning);
+            rObj.SLAResolutionWarning = new Date(rObj.SLAResolutionWarning);
             return rObj;
         });
 
         if (moduleName == 'servicesIncidentTypes' || moduleName == 'categoriesIncidentTypes') {
-            let types = dataArray.map(obj => {
+            let types = dataArray.map((obj, index) => {
```

Comment:
could be simplified using expression: `dataArray.map(({Service, IncidentType, Category}, index) => ({id: index, Service, IncidentType, Category}))`

Reply:
Data map expression simplified